client/src/route/auth/tokenAuth.js

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
+
+const APPLICATION_ID = 'f05709a4c29009cc4bb443c2d13bcd4d9f457107495c5ebae5af8e8a1bed774d';
+const APPLICATION_SECRET = 'e85ae3fb2e7b72d5f709ae2a0753e89e6ddd2beab9042150a0deb8880b1ab489';
+const REDIRECT_URI = 'http://localhost:4000/dashboard';
```

Comment:
all these three variables will have to be in env file

Reply:
This file was redundant so we removed it.